docs/specifications/tests/Consistency-TP/consistency03.md

```diff
@@ -1,61 +1,87 @@
-## CONSISTENCY03: SOA parameters consistency
+# CONSISTENCY03: SOA timers consistency
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY03:** SOA parameters consistency
+**CONSISTENCY03**
 
-### Objective
+## Objective
 
-All SOA record fields except SERIAL, RNAME and MNAME must be consistent
+All SOA record timers fields must be consistent
 across all authoritative name servers.
 
-The inconsistency in these different fields for the designated zone (*Got to
-Verify*) might result in operational inconsistencies.
+The inconsistency in these different fields for the designated zone
+might result in operational inconsistencies.
 
 There are other test cases that provide consistency tests for other
 SOA fields.
 
 
-### Inputs
+## Inputs
 
-The domain name to be tested.
+* The domain name to be tested ("child zone")
 
-### Ordered description of steps to be taken to execute the test case
+## Ordered description of steps to be taken to execute the test case
 
- 1. Obtain a set of name server IP addresses using [Method4] and
-    [Method5].
- 2. Send a SOA query for the zone over UDP to each unique name server
-    IP address.
- 3. Verify that a response is received for each SOA query.
- 4. Collect all SOA records from the answer section of all responses.
- 5. Verify that all SOA records have the same REFRESH value.
- 6. Verify that all SOA records have the same RETRY value.
- 7. Verify that all SOA records have the same EXPIRE value.
- 8. Verify that all SOA records have the same MINIMUM value.
+ 1. Obtain the list of name server IPs for the *child zone* from [Method4] 
+    and [Method5].
 
+ 2. Create an SOA query for *child zone* apex and send it to all name 
+    server IPs.
 
-### Outcome(s)
+ 3. Retrieve the SOA RR from the responses from all name server IPs.
 
-All authoritative name servers must have consistent REFRESH, RETRY,
-EXPIRE and MINIMUM fields.
-If the test does not find any inconsistency, then the test succeeds.
+ 4. If a name server does not respond, emit *[NO_RESPONSE]*.
 
-### Special procedural requirements	
+ 5. If a name server responds but not include a SOA record in the 
+    response, emit *[NO_RESPONSE_SOA_QUERY]*.
+
+ 6. If at least one SOA record has been retrieved, emit 
+    *[ONE_SOA_TIME_PARAMETER_SET]* if all SOA records have 
+    1. the same REFRESH value, 
+    2. the same RETRY value, 
+    3. the same EXPIRE value, and 
+    4. the same MINIMUM value.
```

Comment:
Have you considered the following wording?

> "Emit *[ONE_SOA_TIME_PARAMETER_SET]* if at least one SOA record has been retrieved and all SOA records have:"

Having fewer "if" clauses makes it easier to understand.